modules/cryptofuncs/CMakeLists.txt

```diff
@@ -2,19 +2,16 @@ set (CRYPTOFUNCS_SOURCES
     cryptofuncs.c
 )
 
-find_package(OpenSSL)
 
-if (OPENSSL_FOUND)
-    add_library(cryptofuncs SHARED ${CRYPTOFUNCS_SOURCES})
-    target_include_directories (cryptofuncs PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
-    target_link_libraries(cryptofuncs
-        PRIVATE
-        syslog-ng
-        OpenSSL::SSL
-        OpenSSL::Crypto
-    )
+add_library(cryptofuncs SHARED ${CRYPTOFUNCS_SOURCES})
```

Comment:
Not necessarily in the scope, but with the similar reasoning, the `find_package(OpenSSL)` in afsql can also be removed. And the linking refactored similar way.

There is another detection (call to `find_package(OpenSSL REQUIRED)`) in afsocket, that is rendundant too.

Reply:
Thanks, added a commit that does just that.